src/window.rs

```diff
@@ -78,20 +92,67 @@ impl Window {
         )
     }
 
-    /// Handle Winit window event.
+    /// Create a Verso window with the rendering context.
+    pub fn new_with_compositor(
```

Comment:
this code looks duplicated with `pub fn new(evl: &EventLoopWindowTarget<()>) -> (Self, RenderingContext)`

Reply:
The current limitation is that `RenderingContext` still needs a surface which means it needs an existing window to create. Do you have any suggestions for the abstract?